src/site/content/en/learn-core-web-vitals/lab-and-field-data-differences/index.md

```diff
@@ -183,6 +183,7 @@ element being determined for the same page:
   [text fragment](/text-fragments/#text-fragments), so the LCP element may
   actually be from the middle or bottom of the page (rather than "above the
   fold").
+* Whether a `<video>` element's first frame (not the same as a `poster` image) may be considered an LCP candidate in CrUX, but not in RUM data. The reason for this is that CrUX is not blocked from attributing a page's LCP to a `<video>` embedded in an `<iframe>`—such as with YouTube embeds, for example. RUM data, on the other hand, is gathered through JavaScript, which is unable to access `<iframe>` elements, regardless of whether they are cross-origin or same-origin.
```

Comment:
Rephrasing and adding a link to the changelog. Also generalizing this point to iframes as a whole, and using embedded video as the example.

```suggestion
* Elements within an [`<iframe>`](https://web.dev/crux-and-rum-differences/#iframes) may be reported as the LCP element in CrUX-based field data. For example, as of [August 2023](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/speed/metrics_changelog/lcp.md), Chrome considers the first frame of an embedded autoplaying `<video>` element to be an LCP candidate.
```

Reply:
To be honest, I don't think this article needs to mention this change.

The article's current focus is on the differences between data from a single run (lab) vs. data from a distribution of visits (field) that can be affected by user behavior. It doesn't cover measurement limitations at all.

Barry's article on CrUX vs RUM does cover measurement limitations, so I think it's fine to have it mentioned there.